vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -4069,6 +4445,126 @@ unit:DeciM
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit> ;
   rdfs:label "Decimeter" ;
 .
+unit:DeciM2
+  a qudt:Unit ;
+  qudt:conversionMultiplier 1e-02 ;
+  qudt:conversionOffset 0e+00 ;
+  qudt:description "0,01-fold of the power of the SI base unit metre with the exponent 2" ;
```

Comment:
Would be "0,1-fold..."  if the description is trying to say "one tenth of a Meter, squared"
Or, it is saying "one-hundredth of a square meter", but that isn't a good way to think of it, because the "Deci" applies to the M, and then the result is squared.

Reply:
fixed